var/spack/repos/builtin/packages/py-botocore/package.py

```diff
@@ -12,6 +12,7 @@ class PyBotocore(PythonPackage):
     homepage = "https://github.com/boto/botocore"
     pypi = "botocore/botocore-1.13.44.tar.gz"
 
+    version('1.24.29', sha256='c6f419a79c816297646d228dd07599859df6ee701c6f65a6dd4a35015269eb38')
```

Comment:
Package audit test failed because `depends_on('py-botocore@1.24:',  when='@1.21.28:', type=('build', 'run'))` was added to `py-boto3` without a corresponding 1.24+ here.